chromium_src/chrome/browser/ui/webui/help/version_updater_mac.mm

```diff
@@ -171,7 +171,7 @@ - (void)handleStatusNotification:(NSNotification*)notification {
       break;
 
     default:
-      NOTREACHED_IN_MIGRATION();
+      // We are not currently interested in this status.
```

Comment:
actually it's not really clear what status is assumed and why we are not interested.

Reply:
I guess it's the same value as what happens above for `kAutoupdateAvailable`. But yeah - it's not clear what the default value for `Status` enum is. Would probably be good to VLOG out a warning and what value was received. Something like:
```
VLOG(2) << "Unexpected value sparkle_status=" << sparkle_status << ". Ignoring."
```

Reply:
In our code, we don't use other status.
This status is determined and notified by `SparkleGlue` and `SparkeGlue` only uses above listed status.
So, it's not possible to reach to `default` case.

When I ported to sparkle at first, our status enum was below and all status was handled above.
```
enum AutoupdateStatus {
  kAutoupdateNone = 0,        // no version (initial state only)
  kAutoupdateRegistering,     // no version (asynchronous operation in progress)
  kAutoupdateRegistered,      // no version
  kAutoupdateChecking,        // no version (asynchronous operation in progress)
  kAutoupdateCurrent,         // version of the running application
  kAutoupdateAvailable,       // version of the update that is available
  kAutoupdateInstalling,      // no version (asynchronous operation in progress)
  kAutoupdateInstalled,       // version of the update that was installed
  kAutoupdateRegisterFailed,  // no version
  kAutoupdateCheckFailed,     // no version
  kAutoupdateInstallFailed,   // no version
};
```
and later, we refactored and we shared upstream's keystone glue's AutoupdateStatus.
It's below.
```
enum AutoupdateStatus {
  kAutoupdateNone = 0,        // no version (initial state only)
  kAutoupdateRegistering,     // no version (asynchronous operation in progress)
  kAutoupdateRegistered,      // no version
  kAutoupdateChecking,        // no version (asynchronous operation in progress)
  kAutoupdateCurrent,         // version of the running application
  kAutoupdateAvailable,       // version of the update that is available
  kAutoupdateInstalling,      // no version (asynchronous operation in progress)
  kAutoupdateInstalled,       // version of the update that was installed
  kAutoupdatePromoting,       // no version (asynchronous operation in progress)
  kAutoupdatePromoted,        // no version
  kAutoupdateRegisterFailed,  // no version
  kAutoupdateCheckFailed,     // no version
  kAutoupdateInstallFailed,   // no version
  kAutoupdatePromoteFailed,   // no version
  kAutoupdateNeedsPromotion,  // no version
};
```

Although, keystone glue's AutoupdateStatus includes some other state(`kXXXPromoXX`) that are not handled in our switch statement, we can ignore it as we don't use that status from our `SparkleGlue`.

So, I recommend to update here 
from 
```
      NOTREACHED_IN_MIGRATION();
      // We are not currently interested in this status.
```
to
```
     // We don't use other status.
     NOTREACHED();
```